src/etc/rc.linkup

```diff
@@ -79,6 +79,14 @@ function handle_argument_group($iface, $argument2) {
 		return;
 	}
 
+	/* Ignore events on LAGG members when IP is not configured on it */
+	$lagg_if = link_interface_to_lagg($iface);
+	if (!empty($lagg_if) && empty($ipaddr) && empty($ip6addr)) {
+		log_error("Ignoring link event for LAGG member without IP config");
```

Comment:
"Ignoring link event for LAGG member without IP address configuration"

Reply:
all fixed